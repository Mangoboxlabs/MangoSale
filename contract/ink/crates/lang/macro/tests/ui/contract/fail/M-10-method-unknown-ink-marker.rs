use ink_lang as ink;

#[ink::contract]
mod unknown_method_ink_marker {
    #[ink(storage)]
    pub struct UnknownMethodInkMarker {}

    impl UnknownMethodInkMarker {
        #[ink(constructor)]
        pub fn constructor() -> Self {
            Self {}
        }

        #[ink(message)]
        pub fn message(&self) {}

        #[ink(unknown_marker)]
        pub fn method(&self) {}
    }
}

fn main() {}
