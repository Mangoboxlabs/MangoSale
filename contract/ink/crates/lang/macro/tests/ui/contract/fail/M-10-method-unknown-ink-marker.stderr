error: unknown ink! attribute (path)
  --> $DIR/M-10-method-unknown-ink-marker.rs:17:15
   |
17 |         #[ink(unknown_marker)]
   |               ^^^^^^^^^^^^^^
